test/powershell/Language/Scripting/I18n.Tests.ps1

```diff
@@ -52,6 +52,7 @@ Describe 'Testing of script internationalization' -Tags "CI" {
 
         import-localizedData mydata -uiculture nl-NL -ea SilentlyContinue -ev ev
 
+        $ev | Should Not BeNullOrEmpty
```

Comment:
We should not trust a cmdlet which we check.
If $ev != Null then $ev[0] never raise.
So I suppose `$ev | Should Not BeNullOrEmpty` is correct.